release-notes/release-6.17.0.md

```diff
@@ -0,0 +1,34 @@
+## Release summary
+
+Eclipse Che 6.17 includes:
+
+* **Workspace failure count metric**: Prometheus metric counting the number of workspace failures
```

Comment:
The description here should be from the point of view of user benefit, and should be descriptive: "Keep track of workspace failures" is better. The description should be a sentence: "Added a metric to Prometheus to count the number of workspace failures".